include/item_use.h

```diff
@@ -36,6 +36,8 @@ void ItemUseInBattle_PartyMenuChooseMove(u8 taskId);
 void Task_UseDigEscapeRopeOnField(u8 taskId);
 u8 CanUseDigOrEscapeRopeOnCurMap(void);
 u8 CheckIfItemIsTMHMOrEvolutionStone(u16 itemId);
+void FieldUseFunc_VsSeeker(u8 taskId);
+void Task_ItemUse_CloseMessageBoxAndReturnToField_VsSeeker(u8);
```

Comment:
this should have an argument name since all the others do (even though this is pretty inconsistant across the codebase)